committee_history.md

```diff
@@ -120,3 +120,15 @@
   - Matt Low
   - Olivia MacKenzie-Ross
   - Utkarsh Sharma
+
+## 2022
+- President: Tom Malcom
```

Comment:
```suggestion
Tom Malcolm
```

Reply:
![Screenshot_20211007-211217~2](https://user-images.githubusercontent.com/42644678/136374028-a6161937-79c7-4be8-991b-72d9de412a08.png)
I was merely quoting the primary source.

Reply:
Campbell is also probably misspelled